.eslintrc.json

```diff
@@ -8,21 +8,10 @@
     "sourceType": "module"
   },
   "rules": {
-    "indent": [
-      2,
-      2
-    ],
-    "linebreak-style": [
-      2,
-      "unix"
-    ],
-    "quotes": [
-      2,
-      "double"
-    ],
-    "semi": [
-      2,
-      "always"
-    ]
+    "indent": [2, 2],
+    "linebreak-style": [2, "unix"],
+    "quotes": [2, "double"],
```

Comment:
oh you meant alphabetize it not add http://www.rubydoc.info/gems/rubocop/0.27.0/RuboCop/Cop/Metrics/AbcSize